src/configs/system_config.json

```diff
@@ -1,5 +1,5 @@
 {
     "dataset_path": "/u/abhi24/Datasets/Faces/fairface/",
-    "experiments_folder": "/u/abhi24/Workspace/simba/experiments/",
```

Comment:
This too

Reply:
Are you sure we should remove this system config file too? This is the only example here. Pls let me know! Thx!